packages/frontend/src/components/icons/Arrows.tsx

```diff
@@ -0,0 +1,38 @@
+import cx from 'classnames'
+import React, { SVGAttributes } from 'react'
+
+import { Icon } from './Icon'
+
+export function ArrowUpIcon(props: SVGAttributes<SVGElement>) {
+  return (
+    <ArrowIcon
+      aria-label="Arrow up icon"
+      alt-text="+"
+      className={cx(
+        'inline-block  fill-green-700 dark:fill-green-300',
```

Comment:
```suggestion
        'inline-block fill-green-700 dark:fill-green-300',
```

Removes extra space.